src/IECoreMaya/plugin/Loader.cpp

```diff
@@ -41,6 +41,19 @@
 
 static void *g_libraryHandle = 0;
 
+#define STR2(a) #a
```

Comment:
I'm not happy with these 2 odd STR defines, but I haven't found a better way to do this yet...

Reply:
Not a solution to your STR thing, but...

If I remember correctly, this entire Loader thing is just so we can load manually with RTLD_GLOBAL. We're not quite there in terms of removing that requirement, but it looks as though Maya has supported global loading directly for some time now :

https://mayastation.typepad.com/maya-station/2012/02/global-symbol-evaluation.html

Maybe we can drop the loader entirely?

Reply:
Added todo in f7fcd3bec63d6afd25cf16f688fd80e263ffccfe